recipes/mne/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "mne" %}
+{% set version = "0.13.1" %}
+{% set sha256 = "c9433bbd9dee9bbcddc1b176e127a6e31edc3851f973cb440b35751f6aed42d2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py3k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+  run:
+    - python
+    - matplotlib
+    - nibabel
+    - numpy
+    - scikit-learn
+    - scipy
+    - mayavi
+    - pysurfer
+
+test:
+  imports:
+    - mne
+  commands:
+    - mne
+
+about:
+  home: http://martinos.org/mne
+  license: BSD-3-Clause
+  license_family: BSD
+#  license_file: LICENSE.txt
```

Comment:
Could you please ask upstream to add this to `MANIFEST.in` and xref the issue back here?